packages/suite-desktop/src-electron/modules/tor.ts

```diff
@@ -17,7 +17,8 @@ const load = async ({ mainWindow, store, interceptor }: Dependencies) => {
     const port = await getFreePort();
     const address = `${host}:${port}`;
     const controlPort = await getFreePort();
-    const authFilePath = app.getPath('userData');
+    const userData = app.getPath('userData');
+    const torDataDir = `${userData}/tor-suite-data`;
```

Comment:
```suggestion
    const torDataDir = `${userData}/tor`;
```

Reply:
I wanted to have different than `/tor` in order to make clear that this Tor data directory is for suite app. But I don't really have a hard opinion on this.

Reply:
whole `userData` dir is for suite app - `~/Library/Application\ Support/@trezor/suite-desktop/tor` sounds good to me

Reply:
Done! I also rebased so it does not require manual action to pass the CI from gitlab.